bentoml/cli/serverless.py

```diff
@@ -3,135 +3,73 @@
 import subprocess
 import argparse
 
-from bentoml.utils import Path
-from ruamel.yaml import YAML
+from datetime import datetime
 from packaging import version
 
-AWS_HANDLER_PY_TEMPLATE = """\
-try:
-    import unzip_requirements:
-except ImportError:
-    pass
-import json
-
-import {class_name}
-
-bento_service = {class_name}.load()
-
-def {api_name}(event, context):
-    result = bento_service.{api_name}.handle_aws_lambda_event(event)
-
-    return result
-"""
-
-GOOGLE_MAIN_PY_TEMPLATE = """\
-import {class_name}
-
-bento_service = {class_name}.load()
-
-def {api_name}(request):
-    result = bento_service.{api_name}.handle_request(request)
-    return result
-"""
+from bentoml.cli.whichcraft import which
+from bentoml.cli.aws_lambda_template import AWS_HANDLER_PY_TEMPLATE, \
+     update_serverless_configuration_for_aws
+from bentoml.cli.gcp_function_template import GOOGLE_MAIN_PY_TEMPLATE, \
+     update_serverless_configuration_for_google
 
 default_serverless_parser = argparse.ArgumentParser()
 default_serverless_parser.add_argument('--region')
 default_serverless_parser.add_argument('--stage')
 
+SERVERLESS_PROVIDER = {
+    'aws-lambda': 'aws-python',
```

Comment:
what do you think about making python3 the default? so 'aws-lambda' will match to 'aws-python3', and 'aws-lambda-py2' will match to 'aws-python'